### ISSUE TITLE
Installation Zwave-Adapter auf MacOS via Gui schlägt fehl 

### ISSUE THREAD

**Issue by chris299** [ioBroker/ioBroker.zwave#96]
wenn man den ZWAVE-Adapter via Web-Gui auf MacOS (Catalina) installieren will funktioniert es nicht, wenn iobroker auf dem Mac unter dem Admin-User mit entsprechendem Passwort läuft.
(vermutlich tritt das Problem aber auch in anderen Konstellationen auf)

Das Problem ist das "sudo" im Rahmen des Make-Commandos, weil das nach einer Passwort-Eingabe verlangt, was man per GUI installation aber nicht eingeben kann.
bei der Installation via CLI ist das kein Problem und so funktioniert es dann auch.

**Comment by AlCalzone** [comment]
Warte... Läuft iobroker als admin oder nicht?
Wenn ja, sollte eigentlich das sudo weggelassen werden.

**Comment by chris299** [comment]
ja, iobroker läuft unter dem Admin-User (das ist der einzige User auf dem System, neben "Gast"), der aber natürlich nicht "Admin" heisst...

**Comment by AlCalzone** [comment]
Hmm dann müssen wir nochmal recherchieren wie man den admin auf Mac erkennt
